src/gui/HelpDialog.cpp

```diff
@@ -558,6 +571,10 @@ void HelpDialog::updateAboutText(void) const
 	newHtml += "<li>" + q_("Tester: %1").arg(QString("Khalid AlAjaji")).toHtmlEscaped() + "</li></ul>";
 	newHtml += "<h3>" + q_("Contributors").toHtmlEscaped() + "</h3>";
 	newHtml += "<p>"  + q_("Several people have made contributions to the project and their work has made Stellarium better (sorted alphabetically): %1.").arg(contributors.join(", ")).toHtmlEscaped() + "</p>";
+	newHtml += "<h3>" + q_("Financial contributors").toHtmlEscaped() + "</h3>";
+	newHtml += "<p>"  + q_("Many people and organizations are supporting the development the Stellarium by the money and the best financial contributors (with donations $250+ USD) are %1.").arg(bestFinancialContributors.join(", ")).toHtmlEscaped();
```

Comment:
Many individuals and organizations are supporting the development of Stellarium by donations, and the most generous financial contributors (with donations of $250 or more) are %1.